.ci/lib.sh

```diff
@@ -271,6 +271,27 @@ delete_crio_stale_resource() {
 	sudo rm -rf /etc/systemd/system/crio.service
 }
 
+delete_containerd_cri_stale_resource() {
+	# stop containerd service
+	sudo systemctl stop containerd
+	# remove stale binaries
+	containerd_cri_dir="github.com/containerd/cri"
+	release_dir="${GOPATH}/src/${containerd_cri_dir}/_output/release-stage"
+	binary_dir_union=( "/usr/local/bin" "/usr/local/sbin" )
+	for binary_dir in ${binary_dir_union[@]}
+	do
+		for stale_binary in ${release_dir}/${binary_dir}/*
+		do
+			sudo rm -rf ${binary_dir}/$(basename ${stale_binary})
```

Comment:
If only you could call `sudo make uninstall` ;-)

Reply:
looks to me, `make uninstall` only uninstall `/usr/local/bin/containerd`. 
On AArch64, we built from source code(`make release`), it installed the following binaries:
```
+ install bin/ctr bin/containerd bin/containerd-stress bin/containerd-shim bin/containerd-shim-runc-v1 bin/containerd-shim-runc-v2
./
./usr/
./usr/local/
./usr/local/bin/
./usr/local/bin/ctr
./usr/local/bin/containerd-shim-runc-v1
./usr/local/bin/critest
./usr/local/bin/containerd-stress
./usr/local/bin/containerd-shim-runc-v2
./usr/local/bin/containerd
./usr/local/bin/crictl
./usr/local/bin/containerd-shim
./usr/local/sbin/
./usr/local/sbin/runc

```